mcv/hostname.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+
+def _read_stripped_lines(f):
+    return [l.rstrip() for l in f.readlines()]
+
+
+def _splice_alias(alias, line):
+    """If line is an alias for localhost, append <alias>
+    (if not already present)"""
+    vs = line.split(' ')
+    if vs[0] == '127.0.0.1' and not alias in vs[1:]:
+        vs.append(alias)
+        return ' '.join(vs)
+    else:
+        return line
+
+
+def set_hostname(hostname):
+    """Set the system hostname to <hostname> and add a
+    corresponding entry to /etc/hosts"""
+    os.system('hostname ' + hostname)
+    with open('/etc/hostname', 'w') as f:
+        f.write(hostname)
+    with open('/etc/hosts', 'r+w') as f:
```

Comment:
I don't know, I can put that in but personally I think that's right up against an abstraction/mutability win vs a code noise/boilerplate loss, 4 blocks in a row for the task is kind of lame. will poke around